_includes/nav/current.html

```diff
@@ -6,6 +6,13 @@
   {% assign _current = true %}
 {% elsif page.collection and include.page.subnav.type == page.collection %}
   {% assign _current = true %}
+<!--
+  Use the page's `type` data key to connect the sidenav to a collection.
+  This sets "current" for the pages inside _posts and _security-updates directories.
+  Collections are defined in config.yml, and "posts" is a default collection type in Jekyll.
+-->
+{% elsif page.collection and include.page.type == page.collection %}
+  {% assign _current = true %}
```

Comment:
**suggestion:** Suggesting that we remove the `subnav.type` check here and replace it with the `page.type` check I added in this PR. 

### What it does
The check on line 7 looks for any page  looks for the `subnav.type` data key in the page's front matter and sees if there is a match in the collections defined in [config.yml](https://github.com/uswds/uswds-site/blob/main/_config.yml#L61). If the `subnav.type` matches the collection name, it will mark that page as "current" in the sidenav  for any page that exists inside that collection. 

### Why I want to remove
I want to remove the `subnav.type` check because using `type` inside of the  `subnav` data key seems to conflict with adding link items to the same key `subnav` (meaning, you can't add a subnav type and add a changelog link to the subnav...though I might just be doing it wrong). 

Additionally, `subnav.type` appears to be used in only one other instance on site in [security.md](https://github.com/uswds/uswds-site/blob/main/pages/about/security.md?plain=1#L8). Note that the subnav.type is currently not working on that page because `subnav` is later declared with a changelog link (whoops 😬). 

Converting `subnav.type` to just `type` there  would still attach it to the `security_updates` collection defined in [`config.yml`](https://github.com/uswds/uswds-site/blob/main/_config.yml#L77). 


```suggestion
<!--
  Use the page's `type` data key to connect the sidenav to a collection.
  This sets "current" for the pages inside _posts and _security-updates directories.
  Collections are defined in config.yml, and "posts" is a default collection type in Jekyll.
-->
{% elsif page.collection and include.page.type == page.collection %}
  {% assign _current = true %}
```